locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "Ansible": "Ansible",
+    "Ansible Galaxy/Automation Hub API Token": "Ansible Galaxy/Token de API del concentrador de automatización",
+    "Ansible node instances dedicated for a particular purpose indicated by node type.": "Instancias de nodo Ansible dedicadas a un propósito concreto indicado por el tipo de nodo.",
+    "Ansible works against multiple managed nodes or “hosts” in your infrastructure at the same time, using a list or group of lists known as inventory. Once your inventory is defined, you use patterns to select the hosts or groups you want Ansible to run against.": "Ansible trabaja contra múltiples nodos gestionados o \"hosts\" en su infraestructura al mismo tiempo, mediante una lista o grupo de listas conocidas como inventario. Una vez definido el inventario, puede utilizar patrones para seleccionar los hosts o grupos contra los que desea que se ejecute Ansible.",
+    "API Token": "Token API",
+    "Approvals": "Aprobaciones",
+    "Approve": "Aprobar",
+    "Approve collections": "Aprobar los cobros",
+    "Approved": "Aprobado",
+    "Auditor": "Auditor",
+    "Authentication type": "Tipo de autenticación",
+    "Auto": "Auto",
+    "Automated process cost": "Coste del proceso automatizado",
+    "Automatically enable this rulebook activation to run.": "Habilitar automáticamente la activación de este reglamento para que se ejecute.",
+    "Automation": "Automatización",
+    "Automation Analytics is currently not enabled to send data.": "Actualmente, Automation Analytics no está habilitado para enviar datos.",
+    "Automation Content": "Contenido de automatización",
+    "Automation formula": "Fórmula de automatización",
+    "Automation savings": "Ahorros en automatización",
+    "Automation server": "Servidor de automatización",
+    "Automation servers": "Servidores de automatización",
+    "automation time = successful elapsed total / 3600 automation cost = automation time * configurable automated process cost": "tiempo de automatización = total transcurrido con éxito / 3600 coste de automatización = tiempo de automatización * coste del proceso automatizado configurable",
+    "Automation type": "Tipo de automatización",
+    "AWX": "AWX",
+    "AWX Ansible server": "Servidor AWX Ansible",
+    "Back to Credentials": "Volver a Credenciales",
+    "Back to Jobs": "Volver a Tareas",
+    "Back to Templates": "Volver a Plantillas",
+    "Back to Users": "Volver a Usuarios",
+    "basePathHelpBlock": "basePathHelpBlock",
+    "Being a team member grants the user all the permissions of the team.": "Ser miembro de un equipo otorga al usuario todos los permisos del equipo.",
+    "Branch to use in job run. Project default used if blank. Only allowed if project allow_override field is set to true.": "Rama para utilizar en la ejecución del trabajo. Se utiliza la configuración predeterminada del proyecto si está en blanco. Sólo se permite si el campo allow_override del proyecto está establecido en true.",
```

Comment:
It says: `Se utiliza la configuración predeterminada del proyecto si está en blanco.` But the original string doesn't mention any config, so it looks more like "se utiliza el proyecto predeterminado si está en blanco"